packages/contracts-bedrock/contracts/test/ProxyAdmin.t.sol

```diff
@@ -90,7 +90,7 @@ contract ProxyAdmin_Test is Test {
     function test_proxyType() external {
         assertEq(
             uint256(admin.proxyType(address(proxy))),
-            uint256(ProxyAdmin.ProxyType.OpenZeppelin)
+            uint256(ProxyAdmin.ProxyType.ERC1967)
```

Comment:
I prefer this naming to avoid confusion in the future, as we're not using the exact OpenZeppelin implementation.